components/build/README.md

```diff
@@ -0,0 +1,63 @@
+# `build.rs` utility for Exonum
+
+[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.com/exonum/exonum)
+[![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/blob/master/LICENSE)
+![rust 1.36.0+ required](https://img.shields.io/badge/rust-1.36.0+-blue.svg?label=Required%20Rust)
+
+This crate simplifies writing `build.rs` for Exonum and Exonum services.
```

Comment:
Nit: I'd use "build script" instead of `build.rs`; this [seems to be the common term used in the Rust documentation](https://doc.rust-lang.org/cargo/reference/build-scripts.html).